pkg/logs/auditor/auditor.go

```diff
@@ -193,6 +193,11 @@ func (a *Auditor) updateRegistry(identifier string, offset string) {
 		// specially want to avoid storing the offset
 		return
 	}
+	if _, err := time.Parse(config.DateFormat, offset); err != nil {
```

Comment:
Be careful, the offset can be:
- a cursor for journald
- a real file offset for files
- a timestamp for docker

Tell me if I am wrong but the problem that we are trying to solve is that we don't want to corrupt the registry on file rotation, right ? I think that in such a case, we just want to skip the offset and restart from beginning to collect all logs so I am not sure I would touch the logic here. 

If we we still want to protect the registry, we could for example attach a custom validator to the envelop of each message so that we keep the auditor agnostic:
```go
// from docker
offsetValidator := DockerOffsetValidator
msg := NewMessage(content, DockerOffsetValidator)
// from auditor
if result := msg.validate(); !result.IsValid {
  log.Debugf("updateRegistry error: %v", result.Err)
  return
}
```

Reply:
ok, I will move in the tailer docker the check